defaults/main.yml

```diff
@@ -27,3 +26,7 @@ jenkins_admin_password_file: ""
 jenkins_init_changes:
   - option: "{{ jenkins_java_options_env_var }}"
     value: "{{ jenkins_java_options }}"
+
+jenkins_additional_scripts: True
```

Comment:
Change `jenkins_additional_scripts` to sth like `jenkins_scripts_enable` so variable name will be compatible with varaible from next line.